stock_removal_location_by_priority/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+Stock Removal Location by Priority
+==================================
+
+This module adds a removal priority field on stock locations.
+This priority applies when removing a product from different stock locations
+and the incoming dates are equal in both locations.
+
+Installation
+============
+
+To install this module, you need to do as follows:
+
+#. Go to "Inventory > Configuration > Settings"
+#. In 'Location & Warehouse' section, mark "Use 'Removal Priority' in Locations *"
+
+
+Configuration
+=============
+
+You can configure the removal priority as follows:
+
+#. Go to "Inventory > Configuration > Warehouse Management > Locations"
+#. In each Location form, put a Removal Priority.
+
+
```

Comment:
I would add also the 'usage' section how this behaves and how do you have to configure this priority. 
**tip:** You can try to adapt a little the original issue https://github.com/OCA/stock-logistics-warehouse/issues/286